docs/admin-api-brokers.md

```diff
@@ -87,39 +69,53 @@ Fetch the information of the leader broker, for example, the service url.
 <TabItem value="pulsar-admin">
 
 ```shell
-pulsar-admin brokers leader-broker
+pulsar-admin brokers namespaces use \
+    --url localhost:8080
 ```
 
 Example output:
 
-```json
-{
-  "serviceUrl" : "http://localhost:8080"
-}
+```
+public/default/0x00000000_0x40000000    [broker_assignment=shared is_controlled=false is_active=true]
+public/default/0xc0000000_0xffffffff    [broker_assignment=shared is_controlled=false is_active=true]
+public/functions/0x40000000_0x80000000    [broker_assignment=shared is_controlled=false is_active=true]
+public/functions/0x00000000_0x40000000    [broker_assignment=shared is_controlled=false is_active=true]
+pulsar/standalone/localhost:8080/0x00000000_0xffffffff    [broker_assignment=shared is_controlled=false is_active=true]
+pulsar/localhost:8080/0x00000000_0xffffffff    [broker_assignment=shared is_controlled=false is_active=true]
+public/functions/0x80000000_0xc0000000    [broker_assignment=shared is_controlled=false is_active=true]
+public/default/0x80000000_0xc0000000    [broker_assignment=shared is_controlled=false is_active=true]
 ```
 
 </TabItem>
 <TabItem value="REST API">
 
-{@inject: endpoint|GET|/admin/v2/brokers/leaderBroker|operation/getLeaderBroker?version=@pulsar:version_number@}
+{@inject: endpoint|GET|/admin/v2/brokers/:cluster/:broker/ownedNamespaces|operation/getOwnedNamespaes?version=@pulsar:version_number@}
 
 </TabItem>
 <TabItem value="Java">
 
 ```java
-admin.brokers().getLeaderBroker()
+admin.brokers().getOwnedNamespaces(cluster,brokerUrl);
 ```
 
-For the detail of the code above, see [here](https://github.com/apache/pulsar/blob/master/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BrokersImpl.java#L80)
-
 </TabItem>
 
 </Tabs>
 ````
 
-#### list of namespaces owned by a given broker
+## Update broker conf 
+
+You can update broker configurations using one of the following ways:
 
-It finds all namespaces which are owned and served by a given broker.
+- Supply [configurations](https://pulsar.apache.org/reference/#/@pulsar:version_origin@/config/reference-configuration-broker) when starting up brokers.
```

Comment:
```suggestion
- Supply [configurations](pathname:///reference/#/@pulsar:version_origin@/config/reference-configuration-broker) when starting up brokers.
```

Reply:
If we tweak the heading by adding `dynamically`, then this one can become a supplemented tip.

Reply:
Since the heading is "Update broker conf", so two ways should be treated equally from a doc structure perspective, it does not deserve a tip.

Reply:
This suggestion is given based on the [proposed structure tweak](https://github.com/apache/pulsar-site/pull/493#discussion_r1150112037). 

`Admin-api-brokers` is a task topic for using APIs to implement features around brokers. The link to the reference site supplements the implementation and is somehow different from the primary story. So my idea is we can make it a tip  to provide the other static way when we use `Update broker conf dynamically` as heading 2.